modules/vaos/app/services/vaos/jwt.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
```

Comment:
Is this service document somewhere yet?

Reply:
There's not yet any publicly accessible documentation for VAOS as a whole. The VAOS docs are all in a confluence wiki. This User Service is only for getting a JWT token to connect to the other VAOS endpoints. In follow-up PRs all the endpoints will be feature flagged and won't be turned on in prod until the app is complete. At that time we'll have public documentation in place.